backend/src/main/java/com/bakdata/conquery/resources/api/DatasetResource.java

```diff
@@ -27,4 +30,11 @@ public class DatasetResource extends HDatasets {
 	public FERoot getRoot() {
 		return processor.getRoot(getNamespace().getStorage(), subject);
 	}
+
+	//TODO that slug is pretty big
+	@GET
+	@Path("entity-preview-default-connectors")
```

Comment:
true that
Mach einfach `entity-preview` oder wie du es auch in der QueryResource genannt hast. Wer weiß ob es bei den default Konnektoren bleibt oder ob in Zukunft weitere Konfiguration nötig ist